support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummarySelector.js

```diff
@@ -21,11 +21,15 @@ function mapStateToProps(state: CheckoutState) {
       billingPeriod,
     );
 
-  const digitalBillingPeriod = billingPeriod === 'Annual' ? billingPeriod : 'Monthly';
+  const digitalBillingPeriod = billingPeriod === 'Annual' || billingPeriod === 'Quarterly' ? billingPeriod : 'Monthly';
```

Comment:
`Quarterly` isn't a valid billing period for regular Digital Subscriptions so I think this line should revert to how it was.

Reply:
Very true. Thanks for pointing that out. 👍